conda_forge_tick/status_report.py

```diff
@@ -166,10 +167,15 @@ def graph_migrator_status(
             out["in-pr"].add(node)
             fc = "#31688e"
             fntc = "white"
+            pr_status = pr_json["PR"]['mergeable_state']
+            if pr_status == 'clean':
+                status_icon = ' ✓'
+            else:
+                status_icon = ' ❎'
```

Comment:
should we crib github here? Why is the "x" not red?

Reply:
Red and Green aren't colorblind friendly, and I'm not certain how to turn individual elements of text colors in graphviz

Reply:
ahhh right - good call!